frontend/build-configs/bayern/index.ts

```diff
@@ -86,6 +86,7 @@ let bayern: BuildConfigType = {
     ios: {
         ...bayernCommon,
         bundleIdentifier: "de.nrw.it.ehrensachebayern",
+        provisioningProfileSpecifier: `match AppStore de.nrw.it.ehrensachebayern`,
```

Comment:
```suggestion
        provisioningProfileSpecifier: "match AppStore de.nrw.it.ehrensachebayern",
```
no need for backticks

Reply:
If you feel like it you can add a linting rule for this :)

Reply:
@sarahsporck There is no linting for ts-files set up yet, is it?

Reply:
there are in administration. To include these files you'd need a tsconfig, which includes both projects I think.

Reply:
hmh, not sure if it is good, when administration checks files in frontend. But if you want lint checks for these file, which make sense, maybe we can open a new ticket for this and this of the best solution to realize it.